src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1578,6 +1578,7 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("EXTENDED_SYS_STATE", 1.0f);
 		configure_stream_local("GLOBAL_POSITION_INT", 5.0f);
 		configure_stream_local("GPS2_RAW", 1.0f);
+		configure_stream_local("GPS_GLOBAL_ORIGIN", 0.1f);
```

Comment:
Hi @dagar , did you want this value to be `1.0f` like the others?

Reply:
This message is a bit weird, the mavlink spec says after a new `SET_GPS_GLOBAL_ORIGIN` is received you're supposed to send `GPS_GLOBAL_ORIGIN`. So the idea is to stream by default at an extremely minimal rate, but if `SET_GPS_GLOBAL_ORIGIN` is received another `GPS_GLOBAL_ORIGIN` will go out immediately.

http://mavlink.io/en/messages/common.html#SET_GPS_GLOBAL_ORIGIN